QuestUi/Data/QuestService.cs

```diff
@@ -21,6 +21,7 @@ public async Task Create(string title, string description)
             {
                 Description = description,
                 Title = title,
+                Priority = "Low",

```

Comment:
I'll pass in the actual value once I've implemented my side

Reply:
You write beautiful code :P